pkg/gadgets/trace/open/types/types.go

```diff
@@ -15,20 +15,25 @@
 package types
 
 import (
+	"github.com/kinvolk/inspektor-gadget/pkg/columns"
 	eventtypes "github.com/kinvolk/inspektor-gadget/pkg/types"
 )
 
 type Event struct {
 	eventtypes.Event
 
-	MountNsID uint64 `json:"mountnsid,omitempty"`
-	Pid       uint32 `json:"pid,omitempty"`
-	UID       uint32 `json:"uid,omitempty"`
-	Comm      string `json:"pcomm,omitempty"`
-	Fd        int    `json:"fd,omitempty"`
-	Ret       int    `json:"ret,omitempty"`
-	Err       int    `json:"err,omitempty"`
-	Path      string `json:"path,omitempty"`
+	MountNsID uint64 `json:"mountnsid,omitempty" column:"mntns,width:12,hide"`
+	Pid       uint32 `json:"pid,omitempty" column:"pid,minWidth:7,align:right"`
+	UID       uint32 `json:"uid,omitempty" column:"uid,minWidth:10,align:right,hide"`
+	Comm      string `json:"pcomm,omitempty" column:"comm,maxWidth:16"`
+	Fd        int    `json:"fd,omitempty" column:"fd,width:3,align:right,fixed"`
```

Comment:
I think it should not use fixed. It's not impossible for a process to have more than 999 open files.

Reply:
Mmmm the `minWidth` would be 2 (because of the column name "fd"), and the `maxWidth`, as you said, is 3. Maybe at this point is better to have it fixed to 3?

Reply:
Mauricio meant that `Fd` could be bigger than 999 and therefore the maximum is not 3. Theoretically it could be as big as the `int` that holds it

Reply:
Oh, he said, "It is not impossible" ... @mauriciovasquezbernal, you confused me with this double negation hahahaha